tests/functional/listing/test.js

```diff
@@ -0,0 +1,476 @@
+'use strict';
```

Comment:
I'd move this file to a tests/functional/aws-node-sdk/test/bucket/listingCornerCases.js